bin/pping.py

```diff
@@ -27,29 +27,37 @@
 import signal
 import getopt
 import time
+import logging
 
 import nav.daemon
 from nav.daemon import safesleep as sleep
+from nav.logs import init_generic_logging, convert_debug_level_to_loglevel
 from nav.statemon import statistics
 from nav.statemon import megaping
 from nav.statemon import db
 from nav.statemon import config
 from nav.statemon import circbuf
-from nav.statemon import debug
 from nav.statemon.event import Event
 from nav.statemon.netbox import Netbox
 
 
+LOGGER = logging.getLogger('bin.pping')
+
+
 class pinger:
     def __init__(self, **kwargs):
         signal.signal(signal.SIGHUP, self.signalhandler)
         signal.signal(signal.SIGTERM, self.signalhandler)
         self.config = config.pingconf()
-        debug.setDebugLevel(int(self.config.get("debuglevel", 5)))
-        debug.debug("Setting debuglevel=%s " % self.config.get("debuglevel", 5))
+        debuglevel = int(self.config.get("debuglevel", 5))
```

Comment:
I set debuglevel there in order to shorten the lines before I convert to logging levels.